### DISCUSSION TITLE
eclissi

### DISCUSSION THREAD

**Discussion by aldolit** [Stellarium/stellarium#3416] [Q&A]
vorrei chiedervi come sia possibile determinare e visualizzare la data e l'ora in cui possa avvenire un eclissi tra un pianeta e una stella

**Comment by gzotti** [comment]
> I would like to ask you how it is possible to determine and display the date and time in which an eclipse can occur between a planet and a star

Not in this software.

**Reply by aldolit** [reply]
qualche suggerimento di altro software? Grazie

**Reply by gzotti** [reply]
Google.
